### PR TITLE
fixes some talking related bugs

### PR DESCRIPTION
**Pull request by Casper3667**
fixes https://github.com/Aurorastation/Aurora.3/issues/13054

Also fixes a bug if talking on the department radio with a radio on a slot other than the left ear.
It should now also properly display the correct verb when someone is whispering in another language and if they are singing or not.

### CODE REVIEW

**Comment by Casper3667** [comment]
!review

File: code/modules/mob/living/whisper.dm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	if(client.handle_spam_prevention(message, MUTE_IC))
 		return
 
-	whisper(message)
```
Comment (mikomyazaki) [inline code/modules/mob/living/whisper.dm:17]: Doing this means that whispering will no longer produce messages for eavesdroppers and watchers.
I think you need to find how it is avoiding the part that strips the language prefix in `/mob/living/say` or wherever.

**Review by mikomyazaki — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/whisper.dm (hunk 1 above)
Comment (Casper3667) [reply code/modules/mob/living/whisper.dm:17]: Tested it on a private server and I did not encounter the issues you mentioned, as the ordinary say part seems to take that into account as it runs through the code.

**Review by mikomyazaki — APPROVED** [review]
Seems to work okay. This bit of the code is such a mess :(

**Review by Alberyk — APPROVED** [review]

**Pull request merged**